journalism_workflow/static/journalism_workflow/copy_editor/js/controllers.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('journalism_workflow.copy_editor.controllers')
+    .controller('CopyEditorController', CopyEditorController)
+
+  CopyEditorController.$inject = ['$scope', 'orchestraService'];
+
+  function CopyEditorController($scope, orchestraService) {
+    var vm = $scope;
+
+    // Store the article text document URL for easier summary later
+    var documentCreationStep = orchestraService.taskUtils.findPrerequisite(
```

Comment:
I was hoping your comment here was going to be 'You should have imported the python `find_prerequisite` function.'

Definitely worth an issue--I'll create it.